codec-redis/src/main/java/io/netty/handler/codec/redis/StringRedisMessage.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2016 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License, version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at:
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.netty.handler.codec.redis;
+
+import io.netty.util.internal.ObjectUtil;
+
+public abstract class StringRedisMessage implements RedisMessage {
+
+    protected final byte[] content;
+
+    StringRedisMessage(byte[] content) {
+        ObjectUtil.checkNotNull(content, "content cannot be null");
+        this.content = content;
+    }
+
+    public byte[] content() {
+        return content;
+    }
+
+    public abstract boolean isSuccess();
```

Comment:
`isSuccess` is redundant as the type itself contains the detail whether it's an error (`ErrorRedisMessage`) or something else